crates/store/re_log_encoding/src/codec/file/mod.rs

```diff
@@ -30,6 +30,15 @@ pub(crate) struct MessageHeader {
 }
 
 impl MessageHeader {
+    /// Size of an encoded message header, in bytes.
+    // NOTE: This is `size_of` on a `repr(Rust)` struct,
+    //       which is fine because we control the layout
+    //       in the definition above, and tests would quickly
+    //       catch any sort of regression.
+    pub const SIZE_BYTES: usize = std::mem::size_of::<Self>();
+
+    // NOTE: We use little-endian encoding, because we live in
+    //       the 21st century.
```

Comment:
Reading this just after reviewing an arrow PR that encodes stuff as BE to be spec compliant lol

Reply:
This part is our format, and it doesn't go over the network so nobody can claim it should be network-endian! :smile: